src/components/AboutComponent/AboutComponent.scss

```diff
@@ -0,0 +1,138 @@
+@import "../../styles/utils/variables";
+
+.about {
+  display: flex;
+  justify-content: center;
+  margin-top: 30px;
+
+  .about-layout {
+    text-align: center;
+
+    .about-layout__wrapper {
+      display: inline-flex;
+      justify-content: center;
+      margin: 20px;
+    }
+
+    .about-layout__info {
+      width: 600px;
+      z-index: 1;
+      margin: auto -75px;
+      background-color: $cc-white;
+      border-radius: 5px;
+      padding: 40px;
+      box-shadow: 0 0 20px $cc-purple-dark;
+
+      p {
+        font-size: 1.2rem
+      }
+    }
+
+    .about-layout__foto {
+      width: 600px;
+      height: 400px;
+      background-position: center;
+      background-size: cover;
+
+      &.dollar-foto {
+        background-image: linear-gradient(45deg, transparentize($cc-purple-dark, .5) 5%, transparentize($cc-purple, .5)),
+        url("../../assets/img/currency.jpg");
+      }
+
+      &.bank-foto {
+        background-image: linear-gradient(45deg, transparentize($cc-purple-dark, .5) 5%, transparentize($cc-purple, .5)),
+        url("../../assets/img/bank.jpg");
+      }
+
+      &.rwd-foto {
+        background-image: linear-gradient(45deg, transparentize($cc-purple-dark, .5) 5%, transparentize($cc-purple, .5)),
+        url("../../assets/img/rwd.jpg");
+      }
+    }
+  }
+}
+
+@media screen and (max-width: 1209px) {
+  .about {
+    .about-layout {
+      .about-layout__wrapper {
+        width: 100%;
+        margin: 0 0 20px 0;
+      }
+
+      .about-layout__info {
+        width: 350px;
+        padding: 25px;
+      }
+
+      .about-layout__foto {
+        width: 400px;
+        height: 400px;
+      }
+    }
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .about {
+    .about-layout {
+      .about-layout__info {
+        width: 200px;
+        padding: 20px;
+
+        p {
+          font-size: 0.8rem;
+        }
+      }
+
+      .about-layout__foto {
+        width: 350px;
+        height: 305px;
+      }
+    }
+  }
+}
+
+@media screen and (max-width: 634px) {
+  .about {
+    .about-layout {
+      .about-layout__wrapper {
+        display: flex;
+        flex-direction: column;
+      }
+
+      .about-layout__info {
+        width: 250px;
+        margin: auto auto -140px auto;
+      }
+
+      .second-info {
+        flex-direction: column-reverse;
+      }
+    }
+  }
+}
+
+@media screen and (max-width: 380px) {
+  .about {
+    .about-layout {
+      .about-layout__wrapper {
+        display: flex;
+        flex-direction: column;
+      }
+
+      .about-layout__info {
+        width: 200px;
+      }
+
+      .about-layout__foto {
+        width: 250px;
+        height: 240px;
+      }
+
+      .second-info {
+        flex-direction: column-reverse;
```

Comment:
as above
additionally please consider using `&` symbol. It would be something like this: 
```SCSS
.about-layout__wrapper {
  &.second-info {
    ...
  }
}
```
this way it would be easier to find out that this `second-info` is an modifier of `about-layout-wrapper` element, relations between them. 👍

Reply:
As you mentioned above, I used `nth-of-type` instead of new class

Reply:
thanks bro.